src/powershell/FinOpsToolkit.psm1

```diff
@@ -0,0 +1,274 @@
+data LocalizedData
+{
+    # culture="en-US"
+    ConvertFrom-StringData -StringData @'
+    DeployFinOpsHub = Deploying hub from template '{0}' to resource group '{1}'.
+    ExpandingZip = Expanding zip '{0}'.
+    FoundAsset = Found asset '{0}'.
+    FoundLatestRelease = Found latest release '{0}'.
+    NewDirectory = Creating directory '{0}'.
+    NotLoggedIn = Not logged into Azure. Launching 'Login-AzAccount'.
+    VersionNotFound = Could not find version '{0}' of FinOpsHub. Run 'Get-FinOpsHubVersions' for available versions.
+'@
+}
+
+<#
+    .SYNOPSIS
+        Gets Azure context for logged in user. Will prompt for login if not currently logged in.
+#>
+function Assert-AzContext
```

Comment:
nit: Should we organize functions into regions? Not a big deal. I could see functions for private helpers, generic toolkit, and hubs-specific capabilities, with other specific ones coming later. We can also do this later, of course.

Reply:
done